atom/browser/osr/osr_render_widget_host_view.cc

```diff
@@ -780,14 +727,15 @@ bool OffScreenRenderWidgetHostView::TransformPointToLocalCoordSpace(
     gfx::Point* transformed_point) {
   // Transformations use physical pixels rather than DIP, so conversion
   // is necessary.
-  gfx::Point point_in_pixels = gfx::ConvertPointToPixel(scale_factor_, point);
+  gfx::Point point_in_pixels =
+      gfx::ConvertPointToPixel(current_device_scale_factor_, point);
```

Comment:
I'm confused about when to use `scale_factor_` vs `current_device_scale_factor_`, could you explain the structure to me? Thanks! 😅

Reply:
Okay, so we needed these to be separate before I removed the `ResizeLock`, because there was an async call to `WasResized` when the lock was released, which meant that `scale_factor_` was not always in sync with `current_device_scale_factor_`, which is a `RenderWidgetHostViewBase` field.

Now that the lock is gone, I can merge the two.

Reply:
Great, it seems like that'll be a nice cleanup!